src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -227,50 +244,54 @@ public static AbstractBuilding create(Colony colony, TileEntityColonyBuilding pa
             {
                 BlockPos loc = parent.getPosition();
                 Constructor<?> constructor = oclass.getDeclaredConstructor(Colony.class, BlockPos.class);
-                building = (AbstractBuilding)constructor.newInstance(colony, loc);
-            }
-            else
+                building = (AbstractBuilding) constructor.newInstance(colony, loc);
+            } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)